hetmech/test_short_repeat_error.py

```diff
@@ -0,0 +1,36 @@
+import hetio.readwrite
+import pytest
+
+from .degree_weight import dwpc
+
+
+@pytest.mark.parametrize('error_compound,error_disease,metapath,cypher', [
+    ('DB00860', 'DOID:3083', 'CpDrDdGdD', 0.012299),
+    ('DB00860', 'DOID:3083', 'CpDrDuGuD', 0.006209),
+    ('DB00321', 'DOID:1826', 'CtDrDaGaD', 0.000522),
+    ('DB00945', 'DOID:3393', 'CtDrDdGdD', 0.000000),
+    ('DB00945', 'DOID:3393', 'CtDrDuGuD', 0.000000),
+    ('DB00945', 'DOID:3393', 'CtDrDaGaD', 0.005130),
+    ('DB00945', 'DOID:9008', 'CpDrDuGuD', 0.011211),
+    ('DB00331', 'DOID:11612', 'CtDrDuGuD', 0.002222),
+    ('DB00661', 'DOID:1826', 'CtDrDaGaD', 0.008234),
+    ('DB00571', 'DOID:1826', 'CtDrDaGaD', 0.008234),
+    ('DB00996', 'DOID:1826', 'CtDrDaGaD', 0.000395),
+    ('DB00313', 'DOID:1826', 'CtDrDaGaD', 0.000395)
+])
+def test_short_error(error_compound, error_disease, metapath, cypher):
+    commit = '59c448fd912555f84b9822b4f49b431b696aea15'
+
+    url = (f'https://github.com/dhimmel/hetionet/raw/{commit}/hetnet/json/'
+           f'hetionet-v1.0.json.bz2')
+    graph = hetio.readwrite.read_graph(url)
```

Comment:
Rereading hetionet each time is slow. Even if you fixed this to read hetionet only once, that's still quite a heavy test.

Let's add a `--runslow` pytest option as per https://docs.pytest.org/en/latest/example/simple.html#control-skipping-of-tests-according-to-command-line-option and mark `test_short_error` as slow.